src/classes/HouseholdNamingService.cls

```diff
@@ -227,10 +233,10 @@ public without sharing class HouseholdNamingService {
     //Todo: move to Household Settings?
     public Set<String> getHouseholdNamingContactFields() {
         if (!settings.isAdvancedHouseholdNaming()
-                || householdNamingImpl.setHouseholdNameFieldsOnContact() == null) {
+                || householdNamingImpl.householdNamingFields() == null) {
```

Comment:
Same as initial comment for logical operators.

Reply:
This one looks fine but let me know if Im missing something.